README.md

```diff
@@ -74,15 +78,26 @@ In order to run competition matches you'll need to:
     │   ├── robot_mode.txt
     │   └── worlds
     ├── zone-0
-    |   └── robot.py
+    |   ├── robot.py
+    │   └── log-zone-0-match-<match number>.txt
     └── zone-2
-        └── robot.py
+        ├── robot.py
+        └── log-zone-2-match-<match number>.txt
     ```
 
-4. Load the simulation afresh and let it run. This does not need to be a fresh
-   launch of the Webots simulator program, but does need to be a fresh load of
-   the world.
-   There is a supervisor "robot" in the simulation which will remove any robots
-   for which there is no competitor code and pause the simulation at the end of
-   the match. The only way to restore the robots is to reload the simulation
-   between matches.
+4. Start webots from the command line using:
+
+    ```bash
+    webots --mode=pause
+    ```
+5. Got to File > Make Movie
+   Choose: Resolution: 1920x1080, Quality: 100, Video acceleration: 1.0
+
+6. Press Ctrl + 2 to start simulation
```

Comment:
Perhaps also describe the button, in case the shortcuts differ on different platforms.

Reply:
For me when I start running the video recording it goes to full screen with no way of starting using any on screen buttons

Reply:
It doesn't do that for me. I've no objection to mentioning the shortcut as well.